assets/react/components/MeetingDaysFilterForm.jsx

```diff
@@ -16,7 +16,7 @@ export default class MeetingDaysFilterForm extends Component {
       if (checked) {
         newWeekdayList.push(dayIndex)
       } else {
-        newWeekdayList = _.pull(newWeekdayList, dayIndex);
+        newWeekdayList = pull(newWeekdayList, dayIndex);
```

Comment:
This could also be 
```javascript
newWeekdayList = newWeekdayList.filter( (e,i) => e!==dayIndex );
```
`pull` modifies the array passed in, so
```javascript
pull(newWeekdayList, dayIndex);
```
will be equivalent.